client/jetpack-cloud/sections/partner-portal/credit-card-fields/index.tsx

```diff
@@ -97,8 +103,8 @@ export default function CreditCardFields() {
 				/>
 
 				<SetAsPrimaryPaymentMethod
-					isChecked={ useAsPrimaryPaymentMethod }
-					isDisabled={ isDisabled }
+					isChecked={ useAsPrimaryPaymentMethod || paymentMethods.length === 0 }
```

Comment:
This is more of a question: do we use Yoda conditions in Calypso? 😛

Reply:
Conventions-wise, Yoda conditions are (in my experience) _extremely_ uncommon in the JS community. So far in my work with Calypso, I don't think I've seen many (if any).

Reply:
I think we don't have a strong opinion on this, as we have occurrences of both styles in the codebase:
- `0 === `
- `=== 0`